main.py

```diff
@@ -108,9 +109,14 @@ def save_file():
 if os.path.exists(temp_folder_path):
     shutil.rmtree(temp_folder_path)
 
+
 root = CTk()
+
+screen_width = root.winfo_screenwidth()
+screen_height = root.winfo_screenheight()
+
 root.title("Mouser")
-root.geometry('900x600')
+root.geometry(f"{screen_width}x{screen_height}+0+0")
 root.minsize(900,600)
 
 # Adds menu bar to root and binds the function to file_menu
```

Comment:
Comments are good, help following along with changes and what your adding to the code.